locales/en/fields.json

```diff
@@ -15,7 +15,11 @@
     "hint": "Select the person you are moving"
   },
   "police_national_computer": {
-    "label": "<abbr title=\"Police National Computer\">PNC</abbr> number"
+    "label": "<abbr title=\"Police National Computer\">PNC</abbr> number",
+    "details": {
+      "summaryHtml": "I need to change the PNC number",
+      "html": "If you have created the movement for the wrong person cancel this request and create a new one"
```

Comment:
`movement` -> `move`? We've never referred to it as a "movement" so feels inconsistent. Also, I hate to say it but "movement" reminds of something else which isn't pleasant...

Reply:
That's the copy that @Andy-Millar provided. ¯\_(ツ)_/¯

Reply:
True, but maybe just mention to him and see what he says. I'm pretty sure we had conversations in the early days to make we used `move` instead of `movement` and to try and be consistent.

Reply:
`move` it is